.github/workflows/build.yml

```diff
@@ -374,24 +374,40 @@ jobs:
         with:
           name: moonbeam
           path: build
+      - name: Use pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8.6.12
       - name: Use Node.js 18.x
         uses: actions/setup-node@v3
         with:
           node-version: 20.x
+          cache: "pnpm"
+          cache-dependency-path: test/pnpm-lock.yaml
+      - run: |
+          mkdir -p target/release
+      - name: "Download branch built node"
+        uses: actions/download-artifact@v3.0.2
+        with:
+          name: moonbeam
+          path: target/release
       - name: Get tracing runtimes
         run: |
           ./scripts/build-last-tracing-runtime.sh ${{ needs.set-tags.outputs.git_branch }}
-          mkdir -p archived_tests/moonbase-overrides/
-          mv build/wasm/moonbase-runtime-local-substitute-tracing.wasm archived_tests/moonbase-overrides/
+          mkdir -p test/moonbase-overrides/
+          mv build/wasm/moonbase-runtime-local-substitute-tracing.wasm test/moonbase-overrides/
       - name: Typescript tracing tests (against dev service)
         env:
           BINARY_PATH: ../build/moonbeam
           ETHAPI_CMD: --ethapi=txpool,debug,trace
           FORCE_WASM_EXECUTION: true
           FORCE_COMPILED_WASM: true
           WASM_RUNTIME_OVERRIDES: moonbase-overrides
```

Comment:
remove thesee 5 env var from BINARY_PATH to WASM_RUNTIME_OVERRIDES, they are replaced by the moonwall config